src/System.Reflection.Metadata/tests/TestUtilities/SigningUtilities.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection.Metadata;
+using System.Security.Cryptography;
+
+namespace System.Reflection.PortableExecutable.Tests
+{
+    internal static class SigningUtilities
```

Comment:
I know the light-up will be painful, but is this something that SRM should be able to do on your behalf with a default signature provider?